modules/core/src/main/scala/lucuma/odb/api/model/syntax/OptionalEditorOps.scala

```diff
@@ -7,12 +7,18 @@ import cats.data.State
 import monocle.Optional
 import monocle.state.all._
 
-final class OptionalEditorOps[S, A](self: Optional[S, A]) {
+final class OptionalEditorOps[S, A](val self: Optional[S, A]) extends AnyVal {
```

Comment:
Oops, I didn't mean for this to sneak in but I suppose it is harmless.  I'll be deleting in favor of similar syntax being added to `core` anyway.